examples/01_random_field/03_unstr_srf_export.py

```diff
@@ -7,30 +7,25 @@
 then create a random field at those coordinates.
 """
 import numpy as np
-import matplotlib.pyplot as pt
-from gstools import SRF, Exponential
-from gstools.random import MasterRNG
```

Comment:
Should `MasterRNG` actually be in the `gstools` namespace? - Not so much going on in `gstools.random` and `MasterRNG` is self-explanatory enough to not confuse people.